### PR TITLE
[RDY] Support passing a file containing redis hosts

### PR DESCRIPTION
**Pull request by jburnham**
This allows one to manage a file separately from the service definition
and simply restart the service once the configured file is updated.

### CODE REVIEW

**Comment by oliver006** [comment]
Do you think it would make sense to add a feature to hook into the SIGHUP signal and allow to reload the file list?

File: main.go [hunk 1]
```diff
@@ -54,14 +59,54 @@ func main() {
...
-	for len(passwords) < len(addrs) {
-		passwords = append(passwords, passwords[0])
+	if *redisFile != "" && *redisAddr != "" {
+		log.Fatal("Cannot specify both redis.addr and redis.file")
 	}
-	aliases := strings.Split(*redisAlias, *separator)
-	for len(aliases) < len(addrs) {
-		aliases = append(aliases, aliases[0])
+
+	if *redisFile != "" {
```
Comment (jburnham) [inline main.go]: Yea, I can do this.

**Comment by jburnham** [comment]
@oliver006 SIGHUP support makes sense, but I believe is out of scope for the changes I wanted to make. I'm less comfortable with the changes I think would be necessary to support that logic and it could easily be implemented as a separate PR at a future date if you're ok with that. I have implemented the other suggestions that were made previously and this PR is ready for your review again. Thanks!

**Comment by oliver006** [comment]
👍

**Comment by robbiet480** [comment]
Can this be pushed to Docker Hub? Thanks!

**Comment by oliver006** [comment]
Ah yes, totally forgot about that, done via https://github.com/oliver006/redis_exporter/releases/tag/v0.12

**Pull request merged**